handbook/engineering/web/index.md

```diff
@@ -120,47 +139,53 @@ The meeting notes of web team syncs can be found [in this doc](https://docs.goog
-- [Jean du Plessis](../../../company/team/index.md#jean-du-plessis-he-him) ([Engineering Manager](../roles.md#engineering-manager))
-  - [Felix Becker](../../../company/team/index.md#felix-becker)
-  - [Marek Zaluski](../../../company/team/index.md#marek-zaluski)
-  - [TJ Kandala](../../../company/team/index.md#tharuntej-kandala-he-him)
-  
-_* Joel is the acting PM for this team until we are able to hire a dedicated PM for it. Once that happens, he will focus entirely on code insights._  
-
 ## Growth plan
 
-_Updated 2020-11-18_
+_Updated 2020-01-08*_
+
+*Updated by Joel on behalf of Jean.
 
-We are growing the web team by hiring [frontend engineers](../hiring/software-engineer-frontend.md). When this team gets big enough, we expect to spin off two teams (code insights, and extensions and integrations) and convert the web team into an infrastructure team.
+We are actively hiring [frontend engineers](../hiring/software-engineer-frontend.md) to grow the Web team. 
+When this team team gets big enough, we expect to split into two teams **Frontend platform** and **Extensibility**. We will only make this split once we have identified a dedicated engineering manager and a dedicated product manager.
 
-### Web platform
+_In January 2021, [Code Insights](../code-insights/index.md) moved out of the Web Team into its own team, lead by Felix Becker (Interim EM) and Joel Kwartler (PM)._
 
-This team is accountable for ensuring that our product engineering teams have the tools and components they need to quickly build high quality user experiences everywhere we use web technologies (for example: web application, browser extensions). The primary customers of this team are other engineers at Sourcegraph.
+### Frontend platform: frontend web platform and code browsing 
 
-Examples:
+This team is responsible for:
 
-- Creating a standard component library.
-- Documentation to enable product teams and new hires to quickly learn how we do web development at Sourcegraph.
-- Product improvements that affect our entire app and don't have a clear product team owner (for example: new nav bar, complete site redesign)
-- Efficiency and reliability of frontend CI pipeline steps.
+1. Defining how we do web development at Sourcegraph and ensuring that our product engineering teams have the tools and components they need to quickly build high quality user experiences everywhere we use web technologies (for example: web application, browser extensions).
+1. The core user experience of the Sourcegraph product
 
-### [Code insights](../code-insights/index.md)
+The team's areas of responsibility include:
 
-This team is responsible for building and delivering **code insights** to engineering leaders, empowering data-driven decisions in engineering organizations.
-Learn all about this team on the [code insights team page](../code-insights/index.md).
+- Frontend platform
+   - Creating and maintaining a standard Sourcegraph UI component library.
```

Comment:
My first thought this could mention the aspects/goals of this, like enforcing great UX  throughout our entire web app through abstraction, and aligning the intuitive thing to do for other frontend devs with the right thing to do. But those are probably more goals than "responsibilities".